.github/workflows/gpt-pr-review.yml

```diff
@@ -0,0 +1,26 @@
+on: [pull_request]
+
+jobs:
+  add_pr_comment:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    name: OpenAI PR Comment
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 2
+      - id: review
+        uses: dciborow/action-gpt@dciborow/static-analysis
```

Comment:
@dciborow What does the action do? I don't feel like this is something we should check in this now without having a discussion with the team.

Reply:
this snuck in by mistake from my fork.